src/components/MenuItem/styles.module.css

```diff
@@ -15,4 +16,17 @@
     height: 208px;
     border: solid 1px borderDarkGray;
     background-color: lightGray;
+=======
+  width: 0.5rem;
+  height: 13rem;
+  border: solid 0.0625rem #707070;
+  background-color: #393939;
+}
+
+.regular {
+  width: 218.1875rem;
+  height: 13rem;
+  border: solid 0.0625rem #707070;
+  background-color: #f4f4f4;
+>>>>>>> 6e60ba3ae3467508eafa3c40f2c8c29d47f14a9d
```

Comment:
This looks like it's left over from a resolved merge-conflict, should be removed.